src/pages/Signup/Signup.js

```diff
@@ -15,6 +15,48 @@ export default class Signup extends Component {
     };
   }
 
+  handleJoin = () => {
+    fetch('http://10.58.3.176:8000/users/signup', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.email,
+        password: this.state.password,
+        repassword: this.state.repassword,
+        name: this.state.name,
+        smscheck: this.state.smscheck,
+        emailcheck: this.state.emailcheck,
+      }),
+    })
```

Comment:
```suggestion
  handleJoin = () => {
    const { email, password, repassword, name, smscheck, emailcheck } = this.state
    
    fetch(`${API}/users/signup`, {
      method: 'POST',
      body: JSON.stringify({
        email, password, repassword, name, smscheck, emailcheck,
      }),
    })
```
- api call 하는 ip 주소는 설정 관련된 내용이고 변경이 빈번할 수 있습니다. 만약 api call 하는 포인트가 이곳 하나가 아니라 여러 군데로 늘어났을 때 주소가 변경된다면 그 포인트 수 만큼 돌아다니면서 수정해야 합니다. src/config.js에 상수 변수 하나 선언해두시고 import 하여 사용해주세요.
- 구조분해 할당하시고, 이 경우 객체의 키값과 value에 할당되는 변수명이 같아 축약해줄 수 있습니다.

Reply:
네, 구조분해 할당 및 api 변수 설정 해두었습니다.